migrations/1659684153277-AddGpxFileEntity.ts

```diff
@@ -0,0 +1,26 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class AddGpxFileEntity1659684153277 implements MigrationInterface {
+    name = 'AddGpxFileEntity1659684153277'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`CREATE TABLE "gpx_file" ("id" uuid NOT NULL DEFAULT uuid_generate_v4(), "identifier" character varying NOT NULL, "createdAt" TIMESTAMP NOT NULL DEFAULT now(), "updatedAt" TIMESTAMP NOT NULL DEFAULT now(), "userId" uuid, "tourId" uuid, CONSTRAINT "REL_f781bfb25287d6183600114bb3" UNIQUE ("tourId"), CONSTRAINT "PK_c0679f296e3eddf90376457791b" PRIMARY KEY ("id"))`);
+        await queryRunner.query(`ALTER TABLE "tour" DROP CONSTRAINT "FK_d305ffb20137507c3ac63e128e4"`);
+        await queryRunner.query(`ALTER TABLE "tour" ALTER COLUMN "userId" SET NOT NULL`);
+        await queryRunner.query(`ALTER TABLE "tour" ALTER COLUMN "userId" SET NOT NULL`);
```

Comment:
@Saela Some lines are duplicated. I noticed this before and I don't really now why, but I'm glad you're having this issue as well :P